.github/workflows/code-health.yml

```diff
@@ -50,8 +50,38 @@ jobs:
       - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
       - name: Run ShellCheck
         uses: bewuethr/shellcheck-action@d01912909579c4b1a335828b8fca197fbb8e0aa4
+  check-l1-updated:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
```

Comment:
Is `content: read` enough?  you aren't creating a new PR or committing

Reply:
thanks @andreaangiolillo for your comment. we're creating an artefact with the patch file when there are mutations, don't we need write for that?

Reply:
I don't thinks but I may be wrong, let me know if you found out in the future!